modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -34,6 +34,8 @@ InputParameters validParams<MultiSmoothSuperellipsoidIC>()
   params.addParam<MooseEnum>("check_extremes", extreme_options, "Check all Superellipsoid extremes (center +- each semiaxis) for overlap, must have prevent_overlap set to True.");
   MooseEnum overlap_options("False True", "False");
   params.addParam<MooseEnum>("prevent_overlap", overlap_options, "Check all Superellipsoid centers for overlap with other Superellipsoids.");
+  MooseEnum vary_axes_independently_options("False True", "True");
+  params.addParam<MooseEnum>("vary_axes_independently", vary_axes_independently_options, "If true the length of each semiaxis is randomly chosen within the provided parameters, if false then one random number is generated and applied to all semiaxes.");
```

Comment:
Should i change the other two options i created in previous PR for this IC that made it through to merge but have the same problem?